packages/core/src/cli/lib/shell-helpers.ts

```diff
@@ -0,0 +1 @@
+export const isSubshell = () => (process.env.BASH_SUBSHELL && process.env.BASH_SUBSHELL !== '0') || false;
```

Comment:
simple enough! Any other notes about other things you tried / learned worth adding as comments?

Reply:
I think the _better_ way is probably to verify via that the pid differs from the parent pid, but this is different across shells (zsh for example)